src/app/public/src/modules/help-directive/disable.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, OnInit, OnDestroy } from '@angular/core';
+
+import { HelpWidgetService } from '../shared';
+
+@Directive({
+  selector: '[bbHelp-disableWidget]'
```

Comment:
Selectors should be lower camel-cased: `bbHelpDisableWidget`. https://angular.io/guide/styleguide#style-02-06